docs/aws-cloud9.md

```diff
@@ -236,6 +192,30 @@ Filesystem      Size  Used Avail Use% Mounted on
 /dev/xvda1       20G  9.3G   11G  48% /
 ```
 
+## Install Docker-Compose
+
+By default, Docker is installed on AWS EC2's Ubuntu but Docker-Compose is not, so we have to install it ourselves.
+
+1. Run to download the current stable version of Docker-Compose:
+
+```
+sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
I'm gonna assume that the AWS instance will always be ran with linux in mind and never windows correct?

Reply:
I think that's a safe assumption, yes.

Reply:
For this particular AWS Cloud9 environment, we are using Ubuntu Server 18.04 LTS. When working with AWS EC2, it's common to use the Ubuntu, and Amazon Linux 2 operating systems. As of last year, you can actually use Amazon EC2 Mac Instances to build/test macOS, iOS, iPadOS, tvOS, and watchOS Apps. It was a major breakthrough.